packages/angular/src/app/services/data.service.ts

```diff
@@ -131,4 +132,85 @@ export class DataService {
 
     return from([items])
   };
+
+  public getListDS = () => from([[
```

Comment:
why do you need to dublicate service? can't you just use dx-template-gallery package functions?

Reply:
add dependency for only one service? Besides, it seems we should to use "angular way" services based on rxjs.